src/components/infinite-scroll/InfiniteScroll.tsx

```diff
@@ -93,6 +93,9 @@ function InfiniteScroll({
             container.addEventListener('resize', resizeHandler);
         }
 
+        // calls onContainerScroll until page becomes scrollable
```

Comment:
I think this comment could be a little more explicit about why we need to call onContainerScroll.

maybe `// loads more content until page becomes scrollable` ?